### PR TITLE
update help links to have single html page so we can dynamically

### PR DESCRIPTION
**Pull request by benjamin-craig**
generate help

### CODE REVIEW

File: stix_shifter_modules/aws_athena/configuration/config.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
         },
         "help": {
             "type": "link",
-            "default": "https://www.ibm.com/support/knowledgecenter/SSTDPP_1.6.0/platform/docs/scp-core/data-sources.html"
+            "default": "data-sources-aws.html"
```
Comment (delliott90) [inline stix_shifter_modules/aws_athena/configuration/config.json:8]: Amazon Athena would be a different page from the AWS (Cloud watch logs) page.

File: stix_shifter_modules/cbcloud/configuration/config.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
         },
         "help": {
             "type": "link",
-            "default": "https://www.ibm.com/support/knowledgecenter/SSTDPP_1.6.0/platform/docs/scp-core/data-sources.html"
+            "default": "data-sources-carbon-black.html"
```
Comment (delliott90) [inline stix_shifter_modules/cbcloud/configuration/config.json:18]: Carbon black cloud would be a different page from the CB Response page.

**Comment by benjamin-craig** [comment]
@delliott90 Should I link to the default data sources page data-sources.html for these guys?  Since the help doesn't exist for Carbon Black Cloud or Amazon Athena ?

**Comment by delliott90** [comment]
Until those pages go live, yeah that would probably be best.

**Review by delliott90 — APPROVED** [review]

**Pull request merged**